schemas/com.vimeo/interaction/jsonschema/1-0-0

```diff
@@ -0,0 +1,114 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a Vimeo interactive video hotspot/panel click.",
+  "self": {
+    "vendor": "com.vimeo",
+    "name": "interaction",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "action": {
+      "type": "string",
+      "description": "The action type of the hotspot.",
+      "maxLength": 4096
+    },
+    "actionPreference": {
+      "type": "object",
+      "description": "The user's preferred action type for the hotspot.",
+      "properties": {
+        "pauseOnAction": {
+          "type": [
+            "boolean",
+            "null"
+          ],
+          "description": "Whether to pause when the action type is overlay, url, seek, event, or none."
+        },
+        "overlayId": {
+          "type": [
+            "string",
+            "null"
+          ],
+          "description": "When action is overlay, the displayed unique ID for the overlay when the hotspot is clicked.",
+          "maxLength": 4096
+        },
+        "seekTo": {
+          "type": [
+            "number",
+            "null"
+          ],
+          "description": "When action is seek, the time in seconds that the video should jump to when the hotspot is clicked.",
+          "minimum": 0,
+          "maximum": 2147483647
+        },
+        "seekToFrame": {
+          "type": [
+            "number",
+            "null"
+          ],
+          "description": "When action is seek, the frame that the video should jump to when the hotspot is clicked.",
+          "minimum": 0,
+          "maximum": 2147483647
+        },
+        "url": {
+          "type": [
+            "string",
+            "null"
+          ],
+          "description": "When action is clickthrough, the target URL when the overlay panel is clicked.",
+          "maxLength": 4096
+        }
+      },
+      "additionalProperties": false
+    },
+    "customPayloadData": {
+      "type": [
+        "object",
+        "null"
+      ],
+      "description": "The custom payload data of the interactive hotspot",
+      "additionalProperties": true
+    },
+    "currentTime": {
+      "type": [
+        "number",
+        "null"
+      ],
+      "description": "The current time of the video when the interaction occurs is clicked.",
+      "minimum": 0,
+      "maximum": 2147483647
+    },
+    "hotspotId": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "description": "The unique ID for the hotspot.",
+      "maxLength": 4096
+    },
+    "panelId": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "format": "uuid",
```

Comment:
Just checking, are we sure that this will always be a uuid? Couldn't find the info in the docs, just seems a bit out of place since `hotspotId` and `overlayId` are not uuid just this one.

Reply:
The [example data](https://developer.vimeo.com/player/sdk/reference#interactiveoverlaypanelclicked) for the event showed a UUID, but it's probably not sensible to assume based just on that. Removed.